crates/sequencer/Cargo.toml

```diff
@@ -65,3 +65,4 @@ tempfile = { workspace = true }
 default = []
 local = []
 native = ["citrea-stf/native", "citrea-evm/native"]
+test_mode = []
```

Comment:
Sorry for bikeshedding, I think this should be something like manual_publish_batch/block. Or even with an unsafe_ preffix in order to keep users away from it. Also cargo has `build --all-features` which can trigger it unwillingly. So maybe it would be wisely to put this as an option to sequencer config instead.

Reply:
Hmm, we initially thought of putting this in sequencer config but then decided to go against it.

build --all-features will not cause problems bc everything is branched based on !cfg(test_mode) and cfg(test_mode)

the reason we reverted putting it to config was it looked ugly tbh :D